blog/news-and-updates/2025-01-22-update.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title: Developer weekly update January 22, 2025
+description: This week, we have a new release of Motoko, a forum discussion regarding some changes to dfx, and an NNS proposal that adds 17 new API boundary nodes to the network.
+tags: [Devs]
+image: /img/blog/dev-update-blog-jan-22.jpg
+---
+
+# Developer weekly update January 22, 2025
+
+![January 22, 2025](../../static/img/blog/dev-update-blog-jan-22.jpg)
+
+Hello developers, and welcome to this week's developer weekly update! This week, we have a new release of Motoko, a forum discussion regarding some changes to `dfx`, and an NNS proposal that adds 17 new API boundary nodes to the network. Let's get started!
+
+## Motoko v0.13.6
+
+A new version of Motoko has been released! This release includes an important breaking change regarding orthogonal persistence:
+
+- The Wasm persistence modes used internally for canister upgrades now use lower case names `keep` and `replace` rather than `Keep` and `Replace`. If you are using actor classes with orthogonal persistence, you will need to recompile your code and upgrade it to use the latest `moc` and `dfx`.
```

Comment:
```suggestion
- The Wasm persistence modes used internally for canister upgrades now use lowercase names `keep` and `replace` rather than `Keep` and `Replace`. If you are using actor classes with orthogonal persistence, you will need to recompile your code and upgrade it to use the latest `moc` and `dfx`.
```